functional_testing/Openvcloud/ovc_master_hosted/OVC/c_advanced/machine_tests.py

```diff
@@ -121,7 +120,43 @@ def test003_check_connectivity_through_external_network(self):
         #. Attach VM1 to an external network, should succeed
         #. Assign IP to VM1's external netowrk interface, should succeed.
         #. Check if you can ping VM1 from outside, should succeed
+        #. Check that you can connect to vm with new ip ,should succeed.
+
         """
+        self.lg('%s STARTED' % self._testID)
+        self.lg(" Create a cloudspace CS1, should succeed.")
+        cs1_id = self.cloudapi_cloudspace_create(self.account_id,
+                                                 self.location,
+                                                 self.account_owner)
+
+        self.assertTrue(cs1_id)
+        self.lg("Create VM1,should succeed.")
+        vm1_id = self.cloudapi_create_machine(cloudspace_id=cs1_id)
+        self.assertTrue(vm1_id)
```

Comment:
No need for the this assertion .. the assertion already done inside cloudapi_create_machine